tst/regression/test_suites/output_hdf5/output_hdf5.py

```diff
@@ -81,7 +81,17 @@ def Analyse(self,parameters):
         analyze_status = True
         print(os.getcwd())
 
-        sys.path.insert(1, '../../../../../scripts/python')
+        # Determine path to parthenon installation
+        # Fallback to relative path on failure
+        try:
+            parthenonPath = os.path.realpath(__file__)
+            idx = parthenonPath.rindex('/parthenon/')
+            parthenonPath = parthenonPath[:idx]+'/parthenon'
```

Comment:
I'm not sure... does adding a path object to a string result in a path object? Or does it result in a string? Just to be safe, I'd do
```suggestion
            parthenonPath = os.path.join(parthenonPath[:idx],'parthenon')
```